npm-shrinkwrap.json

```diff
@@ -4450,7 +4450,12 @@
       "version": "1.3.0"
     },
     "wpcom": {
-      "version": "5.3.0"
+      "version": "5.3.0",
+      "dependencies": {
+        "wpcom-xhr-request": {
+          "version": "1.0.0"
```

Comment:
I don't think so since we are defining explicitly the handler passed as an argument of the wpcom instance.

* initial importing: https://github.com/Automattic/wp-calypso/blob/master/client/lib/wpcom-xhr-wrapper/index.js#L4
* request handler when `oauth` is enabled: https://github.com/Automattic/wp-calypso/blob/master/client/lib/wp/browser.js#L21
* wpcom instance: https://github.com/Automattic/wp-calypso/blob/master/client/lib/wp/browser.js#L31